Projects/MouseControl/mouse_control_movement.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+"""
```

Comment:
Comments needed:  Project description, materials needed, where to find the project description, where to find the GoPiGo3.  Best to put in a readme as well with this info.